### PR TITLE
fix using --ids in documentation

### PR DESCRIPTION
**Pull request by HatemAlSum**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

File: src/command_modules/azure-cli-vm/HISTORY.rst [hunk 1]
```diff
@@ -7,9 +7,10 @@ unreleased
 * `vmss get-instance-view`: Fix issue where extra, erroneous information was displayed when using `--instance-id *`
 * `vmss create`: Added support for `--lb-sku`
 * `vm/vmss create`: remove human names from the admin name blacklist
-* `vm/vmss create`: fix issue where the command would throw an error if unable to extract plan information from an image. 
+* `vm/vmss create`: fix issue where the command would throw an error if unable to extract plan information from an image.
 * `vmss create`: fix a crash when create a scaleset with an internal LB
 * `vm availability-set create`: Fix issue where --no-wait argument did not work.
+* Fix documentation with --ids
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: History entries and not desired/required for help text changes.

Comment (HatemAlSum) [reply src/command_modules/azure-cli-vm/HISTORY.rst]: I added it to follow required Pull request guidelines 
`The PR has modified HISTORY.rst with an appropriate description of the change`

should I removed it

Comment (tjprescott) [reply src/command_modules/azure-cli-vm/HISTORY.rst]: Yes, please remove it. I've created a PR to update our PR template :)

**Comment by HatemAlSum** [comment]
I removed changes in rst files

**Pull request merged**